Source/ConversationV1/Conversation.swift

```diff
@@ -43,18 +45,15 @@ public class Conversation {
      - parameter password: The password used to authenticate with the service.
      - parameter version: The release date of the version of the API to use. Specify the date
             in "YYYY-MM-DD" format.
-     - parameter serviceURL: The base URL to use when contacting the service.
      */
     public init(
         username: String,
         password: String,
-        version: String,
-        serviceURL: String = "https://gateway.watsonplatform.net/conversation/api")
+        version: String)
     {
```

Comment:
Yeah, the style for a lot of these inits can be improved now. I'll do that real quick after finishing this review.